# Q4

The bench hold-down clamp is being used to clamp two boards together while they are being glued as shown in Figure Q1(c). The clamp consists of a screw having single square threads of M12 X 2 and coefficient of friction in the screw thread is taken as 0.2.  The screw producing maximum power of 35 W when moving at 2mm/s axially.

 (i)   the torque acting at the screw 

(ii) the compression force exerted on the boards when torque as (i) is applied  

(iii) Recommend the suitable handle length, d if the compression force is increased to 1.5 kN, thus the user need to apply force of 100 N on the handle. 

In [2]:
# Required
from math import sin, cos, tan, pi, atan, asin, degrees, radians

In [3]:
# Dimensions in m
screw_type = 1 # single-start
alpha = 0
alpha_n = 0
d_outer = 0.012
d_collar = 0
pitch = 0.002
f_screw = 0.2
f_collar = 0
screw_power = 35
v_linear = 2
print('Power Screw Type: Square thread')

Power Screw Type: Square thread


(i)   the torque acting at the screw

Power is given, we can find the torque using

$$P=\omega T$$
$$P=\frac {2\pi n_{rpm}}{60} T$$

Find $n_{rpm}$ using screw Linear Velocity

$$v_{screw}=Lead\times \frac{n_{rpm}}{60}$$

In [4]:
Lead = screw_type*pitch
print(f'Lead={Lead*1000}mm/turn')

Lead=2.0mm/turn


$$2=2\times \frac{n_{rpm}}{60}$$

In [5]:
n_rpm = 60
print(f'n_rpm={n_rpm}rpm')

n_rpm=60rpm


$$35=\frac {2\pi (60)}{60} T$$

In [6]:
Torque = screw_power*60/(2*pi*n_rpm)
print(f'Torque, T = {Torque:.3f}Nm')

Torque, T = 5.570Nm


(ii) the compression force exerted on the boards when torque as (i) is applied

Free body diagram

$$+\circlearrowleft \sum M_B=0$$
$$-F_A(80)+F_C(300)=0$$

Find force at power screw to find $F_c$
Force at power screw is load, W

$$T_u = \frac{Wd_m}{2} \frac{f+\cos\alpha_n \tan\lambda}{\cos\alpha_n + f\tan\lambda} + \frac{Wf_c d_c}{2}$$

No collar friction and square thread is used

$f_c = 0$ and $\alpha=0 , \alpha_n=0$

$$T_u = \frac{Wd_m}{2} \frac{f+\cos(0) \tan\lambda}{\cos(0) + f\tan\lambda}$$

Find mean diameter, $d_m = \frac {D+(D-p)}{2}$

$d_m = \frac {12+(12-2)}{2}$

In [7]:
d_inner = d_outer-pitch
d_mean = (d_outer+d_inner)/2

print(f'Mean diameter={d_mean}m')
print(f'Inner diameter={d_inner}m')

Mean diameter=0.011m
Inner diameter=0.01m


From equation
$$ \tan\lambda = \frac{L}{\pi d_m} $$

In [8]:
tan_lambda = Lead/(pi*d_mean)
print(f'tan(lambda)={tan_lambda}')
lambda_helix = degrees(atan(tan_lambda))
print(f'Lambda = {lambda_helix}')

tan(lambda)=0.057874524760689224
Lambda = 3.312271200129051


$$5.57 = \frac{F_A (0.011)}{2} \frac{(0.2)+\cos(0) \tan\lambda}{\cos(0) - (0.2)\tan\lambda}$$

In [9]:
F_a = 2*(cos(radians(alpha_n))-(f_screw*tan(radians(lambda_helix))))*Torque/(d_mean*(f_screw+(cos(radians(alpha_n))*tan(radians(lambda_helix)))))
print(f'F_a={F_a:.3f}N')

F_a=3882.047N


From
$$-F_A(80)+F_C(300)=0$$
$$-(3882)(80)+F_C(300)=0$$

In [10]:
F_c = round(F_a*80/300,2)
print(f'F_c = {F_c} N')

F_c = 1035.21 N


In [11]:
theta = degrees(asin(100/300))
print(f'theta={theta}')

theta=19.47122063449069


In [12]:
F_cy=F_c/cos(radians(theta))
print(f'F_cy={F_cy:.3f}N')

F_cy=1098.006N


(iii) Recommend the suitable handle length, d if the compression force is increased to 1.5 kN, thus the user need to apply force of 100 N on the handle. 

In [13]:
F_cy = 1500
F_c = F_cy*cos(radians(theta))
print(f'New perpendicular force at C, Fc={F_c:.3f}N')

New perpendicular force at C, Fc=1414.214N


From
$$-F_A(80)+F_C(300)=0$$
$$-F_A(80)+(1414.21)(300)=0$$

In [14]:
F_a=F_c*300/80
print(f'New  force at A, Fa={F_a:.3f}N')

New  force at A, Fa=5303.301N


In [15]:
T_u =(F_a*d_mean/2)*((f_screw+(cos(radians(alpha_n))*tan(radians(lambda_helix))))/(cos(radians(alpha_n))-(f_screw*tan(radians(lambda_helix)))))+(F_a*f_collar*d_collar/2)
print(f'T_u={T_u:.2f}N')

T_u=7.61N


Torque equation
$$T_u = r \times F_a$$
$$7.61 = r \times 5303.3$$
$$d=2\times r$$

In [16]:
d_forLever = 2*T_u/F_a
print(f'Lever length, d={d_forLever}m')

Lever length, d=0.0028698378729753162m
